libs/Z2AudioLib/Z2SoundHandles.cpp

```diff
@@ -1,57 +1,63 @@
 /* Z2SoundHandles.cpp autogenerated by split.py v0.3 at 2021-01-05 01:21:30.657620 */
 
+#include "Z2AudioLib/Z2SoundHandles/Z2SoundHandles.h"
 #include "global.h"
 
-extern "C" {
-// __ct__14Z2SoundHandlesFv
-// Z2SoundHandles::Z2SoundHandles(void)
-asm void Z2SoundHandles(void) {
-    nofralloc
-#include "Z2AudioLib/Z2SoundHandles/asm/func_802AB07C.s"
+Z2SoundHandles::Z2SoundHandles() {
+    mNumHandles = 0;
 }
 
-// __dt__14Z2SoundHandlesFv
-// Z2SoundHandles::~Z2SoundHandles(void)
-asm void Z2SoundHandles_NS_dtor(void) {
-    nofralloc
-#include "Z2AudioLib/Z2SoundHandles/asm/func_802AB0B4.s"
+Z2SoundHandles::~Z2SoundHandles() {
+    this->deleteHandlesPool();
 }
 
-// initHandlesPool__14Z2SoundHandlesFUc
-// Z2SoundHandles::initHandlesPool(unsigned char)
-asm void Z2SoundHandles_NS_initHandlesPool(void) {
-    nofralloc
-#include "Z2AudioLib/Z2SoundHandles/asm/func_802AB118.s"
+void Z2SoundHandles::initHandlesPool(u8 pNumHandles) {
+    mNumHandles = pNumHandles;
 }
 
-// deleteHandlesPool__14Z2SoundHandlesFv
-// Z2SoundHandles::deleteHandlesPool(void)
-asm void Z2SoundHandles_NS_deleteHandlesPool(void) {
+asm void Z2SoundHandles::deleteHandlesPool() {
     nofralloc
 #include "Z2AudioLib/Z2SoundHandles/asm/func_802AB120.s"
 }
 
+extern "C" {
 // __dt__31JASMemPool<17Z2SoundHandlePool>Fv
 // JASMemPool<17Z2SoundHandlePool>::~JASMemPool<17Z2SoundHandlePool>(void)
 asm void JASMemPool_NS_dtor_X4_(void) {
     nofralloc
 #include "Z2AudioLib/Z2SoundHandles/asm/func_802AB200.s"
 }
+};
 
-// getHandleSoundID__14Z2SoundHandlesF10JAISoundID
-// Z2SoundHandles::getHandleSoundID(JAISoundID)
-asm void Z2SoundHandles_NS_getHandleSoundID(void) {
-    nofralloc
-#include "Z2AudioLib/Z2SoundHandles/asm/func_802AB254.s"
+JAISoundHandle* Z2SoundHandles::getHandleSoundID(JAISoundID pSoundId) {
+    for (JSULink<Z2SoundHandlePool>* link = this->getFirst(); link != NULL;
```

Comment:
```suggestion
    JSULink<Z2SoundHandlePool>* link;
    for (link = this->getFirst(); link != NULL;
```
is it possible to put the declaration on another line so clang-format doesnt multiline the loop statements?